### DISCUSSION TITLE
Idea from reddit user: yournonstoplover

### DISCUSSION THREAD

**Discussion by aklidata** [PullDating/Org#81] [Ideas & Features]
For context, I'm a man seeking a woman to be in a potentially serious, monogamous relationship with.

People that are looking for a serious relationship or marriage, should be only shown a pool of people that also want a serious relationship or marriage. People looking for casual, hookups, or not sure, should be in their own space together, far away from the serious relationship or marriage seekers. People that prefer not to say what they want can enjoy their own company with other secret agents and spies, holding secrets.

Ghosters should be penalized, such as their profile be demoted and shown less. I'm only brainstorming here, but if a person doesn't respond within 24 or 48 hours, they would receive some negative point, which would be calculated on their profile as a percentage the person likely to ghost. Obviously, people that are very busy, will not like this idea, but if they are so busy, why are they even online dating? Manage your time better. Coffee Meets Bagel does have something like this, where the person's chats with matches is shown as a percentage. But that's all it is though.

Hinge, and I believe Bumble too, had a dealbreaker toggle for preferences. Therefore the app would not show people outside of the preference. For example, a preference for non-smokers, and setting the dealbreaker toggle, prevented showing smokers of any kind. I really liked that feature. I wish other dating apps had this.

I hate low-effort people. Especially after matching with a woman and she barely musters the energy to type a succinct sentence. I had matched with women that I had an actual conversation with and -- gasp -- asked me questions too. Most of them I went on a date or two with as well. But such women are very few unfortunately. I really wish there was a way to filter out the low-effort people on dating apps. But I honestly don't have a viable idea for managing them. Maybe a rating system for the person's conversation?

Filling out at least 3 meaningful prompts on a profile should be mandatory. Such as who they are, what they like, and what they are looking for in a romantic partner. This would eliminate the people that have blank profiles. Not sure if those are bots or just very lazy people.

Profiles that have not been active for 48-72 hours, or more, should be demoted and shown less. And a clear indicator should be present on their profile notifying the viewer that the profile has been inactive. Also, any profile that hasn't seen activity in like a week or two, should be automatically purged.

**Comment by WilliamDormer** [comment]
Definitely should be a ghosting penalty. Either unmatch or continue to date them.

**Comment by WilliamDormer** [comment]
I like the idea of a "soft filter" or a sliding scale where you can mostly filter it out, but leave a little bit of wiggle room.
